data/json/items/tool/fire.json

```diff
@@ -17,5 +82,196 @@
+        "active": true,
+        "need_charges": 1,
+        "need_charges_msg": "Nothing happens.",
+        "menu_text": "Light up",
+        "type": "transform"
+      },
+      { "type": "firestarter", "moves": 50 }
+    ],
+    "flags": [ "FIRESTARTER" ]
+  },
+  {
+    "id": "ref_lighter_on",
+    "type": "TOOL",
+    "name": "refillable lighter",
+    "name_plural": "refillable lighters",
+    "description": "This is a lighter featuring a flip top cover and refuelable tank.  It is lit.",
+    "weight": "20 g",
+    "volume": "31 ml",
+    "price": 3000,
+    "material": "aluminum",
+    "symbol": ",",
+    "color": "light_gray",
+    "ammo": "gasoline",
+    "initial_charges": 25,
+    "max_charges": 50,
+    "charges_per_use": 1,
+    "turns_per_charge": 20,
+    "revert_to": "ref_lighter",
+    "use_action": [
+      { "type": "firestarter", "moves": 40, "moves_slow": 500 },
+      {
+        "target": "ref_lighter",
+        "msg": "You extinguish the lighter.",
+        "active": false,
+        "menu_text": "Extinguish",
+        "type": "transform"
+      }
+    ],
+    "flags": [ "FIRE", "LIGHT_8", "FLAMING", "REFILLABLE_LIGHTER", "TRADER_AVOID", "WATER_EXTINGUISH" ]
+  },
+  {
+    "id": "tinderbox",
+    "type": "TOOL",
+    "name": "ember carrier",
+    "description": "This is a small container for holding tinder, with holes punched in it to provide a controlled airflow.  It can nurture a smoldering ember for a long time, to start fires without modern tools.",
+    "weight": "500 g",
+    "volume": "500 ml",
+    "price": 1000,
+    "material": "steel",
+    "symbol": ",",
+    "color": "light_gray",
+    "ammo": "tinder",
+    "max_charges": 100,
+    "use_action": {
+      "target": "tinderbox_on",
+      "msg": "You light the tinder.",
+      "active": true,
+      "need_fire": 1,
+      "need_fire_msg": "You need a lighter or fire to light this.",
+      "need_charges": 1,
+      "need_charges_msg": "The ember carrier is out of tinder.",
+      "type": "transform"
+    }
+  },
+  {
+    "id": "tinderbox_on",
+    "type": "TOOL",
+    "name": "ember carrier (lit)",
+    "name_plural": "ember carriers (lit)",
+    "description": "This is a small container for holding tinder, with holes punched in it to provide a controlled airflow.  An ember is smoldering inside it, and can be used to start a campfire.",
+    "weight": "500 g",
+    "volume": "500 ml",
+    "price": 1000,
+    "material": "steel",
+    "symbol": ",",
+    "color": "light_gray",
+    "ammo": "tinder",
+    "max_charges": 100,
+    "turns_per_charge": 600,
+    "revert_to": "tinderbox",
+    "use_action": [
+      { "type": "firestarter", "moves": 200, "moves_slow": 2000 },
+      {
+        "target": "tinderbox",
+        "msg": "The ember is extinguished.",
+        "active": false,
+        "menu_text": "Extinguish",
+        "type": "transform"
+      }
+    ],
+    "flags": [ "FIRESTARTER" ]
+  },
+  {
+    "id": "whistle_multitool",
```

Comment:
This doesn't belong in fire, that's definitely not this tool's primary function

Reply:
I renamed funnel.json to raincatchers.json. Put the whistle multitool back to tools.json. I will move it to misc.json when we get there.

Resolved merge conflicts.